### ISSUE TITLE
Missing .gitignore

### ISSUE THREAD

**Issue by jboxman** [Kornil/simple-react-app#12]
I just installed this globally and ran the setup script:

```
simple-react-app sra-test
Initializing project..
npm init -- done

Installing deps -- it might take a few minutes..
events.js:163
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open '/Users/jasonb/.config/fnm/lib/node_modules/simple-react-app/.gitignore'
```

I can confirm I have no .gitignore file in that directory.

**Comment by Kornil** [comment]
checking, mind sharing your OS @jboxman ?

**Comment by Kornil** [comment]
@smalluban I hotfixed this half an hour ago (added .npmignore empty) could you try `npm i -g simple-react-app` again? I'm not quite sure this is enough to fix it tho.

**Comment by Kornil** [comment]
@jboxman @smalluban bug has been fixed here:

removed faulty code https://github.com/Kornil/simple-react-app/commit/8e37fc65340f58c3970bf72e74355840504c72ca
and enabled new copy for `.gitignore` here https://github.com/Kornil/simple-react-app/commit/8c60e1e0b1d5183ac42e2d2a21a862699e4b9527 

Explanation of the bug:
Some OSes (manjaro was golden, could never replicate it) when npm installing module, strip the `.gitignore` file, since there were instructions on the code to copy it over it was failing cause there was no source to copy from.

To solve the issue now we get the file from an `https.get` request to this repository and then create it with the fetched data.

Fixed on `1.2.2`
`npm i -g simple-react-app` again and it should now work flawlessy.
